### PR TITLE
package.json license change

### PR DESCRIPTION
**Pull request by rajikaimal**
* change license to MIT
* change author to 99xtechnology.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,8 @@
     "edit",
     "webpack"
   ],
-  
   "author": "99xTechShift",
```
Comment (rehrumesh) [inline package.json]: In other projects we used "99xtechnology.com"
@charithsoori any ideas?

Comment (rajikaimal) [inline package.json]: In most of the repos author is '99xtechnology.com', shall I change it ?

Comment (charithsoori) [inline package.json]: Please Change it  :)

Comment (rajikaimal) [inline package.json]: Changed author :tada:

**Pull request merged**